build-tools/xaprepare/xaprepare/Application/AndroidPlatform.cs

```diff
@@ -6,36 +6,40 @@ namespace Xamarin.Android.Prepare
 {
 	class AndroidPlatform
 	{
+		public string ApiName    { get; }
 		public uint ApiLevel     { get; }
 		public string PlatformID { get; }
 		public string Framework  { get; }
 		public bool Stable       { get; }
 		public bool Supported    { get; }
+		public string Include    { get; }
```

Comment:
Given that the `Include` value *is* the `Framework` value -- when specified -- why introduce `Include` in the first place?

Just use the `Framework` value in the `<AndroidApiInfo Include="{androidPlatform.Framework}">`.

Reply:
Hi @jonpryor, I tried to do that, but it seems the framework field is being used for other purposes. When I added the missing framework versions I got all sorts of different errors and was unable build. Maybe @grendello knows the reason and has a suggestion on how I could fix it to use the framework and avoid the new field.

Reply:
`Framework` corresponds to the old makefile `FRAMEWORKS` (and friends) variable - it should only be used when the framework in question is supported by us and the new `Include` property appears to extend beyond that set.

Reply:
@jonpryor do you have any other concerns?